frontend/src/types.ts

```diff
@@ -508,6 +508,7 @@ export interface TeamBasicType extends WithAccessControl {
     organization: string // Organization ID
     project_id: number
     api_token: string
+    secret_api_token: string
```

Comment:
I like `secret_api_token` btw; `flag_definitions_api_token` might be a bit more specific but it's also less obvious that it's a secret.

Reply:
👍 But in the UI, you’re ok with it being a Flag Definitions token or should I change it there too?

Reply:
Thanks for the feedback @benjackwhite 

What are your thoughts on the fact this PR removes the ability to rotate keys like we had with personal api tokens for local evaluation? Not a concern? Minor concern? Or Major concern?

Reply:
> We don't need to solve every problem at once so it would be a good follow up rather than making this really big as a change.

Understood.

> If you have capacity it would be a great follow up and as I say it would be good to have the same thing for normal project keys (although that's probably the part that makes it way more effort)

Yeah, normal project keys will be out of scope of my changes. 😆 

However, one thing I might look at is making this column an array column so that it's not hard to follow-up.

Reply:
Ok, nevermind. A migration to an array column isn't too hard. Just create a new column (with a plural name), then copy data into it, and be done.